src/components/composer/composer/composer.ts

```diff
@@ -86,6 +86,17 @@ css/* scss */ `
       }
     }
 
+    .o-composer[contenteditable="true"]:empty::before {
```

Comment:
```suggestion
    .o-composer[contenteditable="true"]:empty:not(:focus)::before {
```

that way you don't need the next css rule with `display: none;`

Reply:
Ok, thanks for the suggestion! Changes made